sherlock/config/default_config.yaml

```
@@ -474,7 +480,9 @@ firecloudAccountManager:
   #     - "user-to-never-disable@test.firecloud.org"
   #   # How long to allow a new account to exist without a login before it is disabled. We set this low
   #   # to help make sure the invite emails Google Workspace sends aren't a security risk.
+  #   # The largest unit that can be passed here is hours (so it's unambiguous about leap sends etc.)
```

```suggestion
  #   # The largest unit that can be passed here is hours (so it's unambiguous about leap secs etc.)
```
Wondering if this is a typo, not sure what 'leap sends' are 🤔

---

Good find, thanks